app/styles/_uswds-theme-custom-styles.scss

```diff
@@ -28,37 +28,6 @@ i.e.
   flex-direction: column;
 }
 
-.header {
-  align-items: center;
-  border-bottom: 1px solid #eaeaea;
-  display: flex;
-  justify-content: center;
-  padding: 2rem;
-  position: sticky;
-  top: 0;
-}
-
-.footer {
-  align-items: center;
-  background-color: rgb(250, 250, 250);
-  border-top: 1px solid #eaeaea;
-  bottom: 0;
-  justify-content: center;
-  padding: 2rem;
-  position: sticky;
-  display: flex;
-  width: 100%;
-}
-
-.main {
-  align-items: center;
-  background: linear-gradient(white, rgb(237, 234, 244));
-  display: flex;
-  flex: 1;
-  justify-content: center;
-  padding: 2rem;
-}
-
 .title a {
```

Comment:
nit: we can remove these styles and the `className` in `index.tsx`

Reply:
Done, thanks for the tip on the snapshot!